src/helpers/EmailNotificationAPI.js

```diff
@@ -27,15 +27,11 @@ class EmailNotificationAPI {
    */
   static transportCreator() {
     const {
-      EMAILHOST, EMAILPORT, EMAILUSER, EMAILPASS
+      EMAILUSER, EMAILPASS
     } = process.env;
-    if (!EMAILUSER || !EMAILPASS || !EMAILHOST) {
```

Comment:
Why did you remove this check?

Reply:
There was no coverage on this line because they are environment variables and such will always be set by us

Reply:
But what happens when I forgot to set them?